src/common/collection_column_specs/genome_attribs-GTDB.yml

```diff
@@ -49,7 +49,7 @@ columns:
 
    - key: classification  # GTDB classification, might need better tokenizing
      type: string
-     filter_strategy: fulltext
```

Comment:
Do we need to update genome_attribs-GROW/ENIGMA.yml as well?

Reply:
ENIGMA and GROW are just softlinks to PMI

Reply:
I see.